torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6550,6 +6592,42 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestGradients', 'test_fn_grad'),
                SkipInfo('TestGradients', 'test_fn_gradgrad'),
                SkipInfo('TestGradients', 'test_forward_mode_AD'))),
+    OpInfo('meshgrid',
+           variant_test_name='variadic_tensors',
+           # Our implementation corresponds to "ij" indexing for
+           # numpy.meshgrid, but its default value is "xy".
+           ref=lambda *tensors: np.meshgrid(*tensors, indexing='ij'),
+           dtypes=all_types_and_complex_and(torch.bfloat16, torch.bool, torch.float16),
+           sample_inputs_func=sample_inputs_meshgrid('variadic'),
```

Comment:
Looking at this I understand why the sample inputs function returns a callable, but a simpler pattern is just to partial a sample inputs function like `div` does:

https://github.com/pytorch/pytorch/blob/c5de83adca05f771c07c4aa8c2534517e0050b32/torch/testing/_internal/common_methods_invocations.py#L5777

Reply:
done! nice!